systest/acl/restore/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       source: ./acl-secret
       target: /secret/hmac
       read_only: true
-    command: /gobin/dgraph alpha --my=alpha1:7080 --zero=zero1:5080
+    command: dgraph alpha --my=alpha1:7080 --zero=zero1:5080
```

Comment:
Can the above /gobin bind point be removed?

Reply:
I will resolve these conversations and we can continue discussion on the comment above